assets/i18n/en.json

```diff
@@ -551,6 +552,7 @@
     "I18N_TOPNAV_ABOUT_OPPIA": "About Oppia",
     "I18N_TOPNAV_ADMIN_PAGE": "Admin Page",
     "I18N_TOPNAV_BLOG": "Blog",
+    "I18N_TOPNAV_CONTACT" : "Contact",
     "I18N_TOPNAV_CONTACT_US": "Contact Us",
```

Comment:
Can we use or remove this? It doesn't seem to be used anywhere.

Reply:
it is required because in the file it is refrencing.

Reply:
What file?

Reply:
![mcmc](https://user-images.githubusercontent.com/45730556/73762767-947b5a80-4796-11ea-89dd-1c3ea7676a84.png)

I18N_TOPNAV_CONTACT is referenced there.

Reply:
I'm referencing to `I18N_FOOTER_CONTACT_US`, sorry forgot to explain that before.